docs/AboutUs.md

```diff
@@ -55,4 +55,4 @@ We are a team based in the [School of Computing, National University of Singapor
 [[portfolio](team/kennycjy.md)]
 
 * Role: Developer
-* Responsibilities: TBD
+* Responsibilities: Entire tagging/untagging functionality
```

Comment:
"Entire" might not be the best word for this. But it's fine.

Reply:
let kenny change himself